diff -uNr ansible-runner-2.4.0/pyproject.toml ansible-runner-2.4.0-new/pyproject.toml
--- ansible-runner-2.4.0/pyproject.toml	2024-05-16 19:49:30.000000000 +0200
+++ ansible-runner-2.4.0-new/pyproject.toml	2025-03-03 01:12:41.489675548 +0100
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=45, <=69.0.2", "setuptools-scm[toml]>=6.2, <=8.0.4"]  # pin max versions of build deps and update as needed
+requires = ["setuptools>=45, <=69.0.2", "setuptools-scm[toml]>=6.2, <=8.2.0"]  # pin max versions of build deps and update as needed
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
