diff -uNr autocommand-2.2.2/pyproject.toml autocommand-2.2.2-fix/pyproject.toml
--- autocommand-2.2.2/pyproject.toml	2022-11-18 20:13:23.000000000 +0100
+++ autocommand-2.2.2-fix/pyproject.toml	2024-01-02 00:31:08.430800713 +0100
@@ -4,6 +4,7 @@
 authors = [
   { name="Nathan West" },
 ]
+license = "dynamic"
 description = "A library to create a command-line program from a function"
 readme = "README.md"
 requires-python = ">=3.7"
