diff -uNr autocommand-2.2.2/pyproject.toml autocommand-2.2.2-fix/pyproject.toml
--- autocommand-2.2.2/pyproject.toml	2022-11-18 20:13:23.000000000 +0100
+++ autocommand-2.2.2-fix/pyproject.toml	2024-01-02 00:29:06.095186274 +0100
@@ -1,6 +1,7 @@
 [project]
 name = "autocommand"
 version = "2.2.2"
+license = dynamic
 authors = [
   { name="Nathan West" },
 ]
@@ -22,3 +23,7 @@
 [project.urls]
 "Homepage" = "https://github.com/Lucretiel/autocommand"
 "Bug Tracker" = "https://github.com/Lucretiel/autocommand/issues"
+
+[build-system]
+requires = ["setuptools>=49.2.0"]
+build-backend = "setuptools.build_meta"
