diff -uNr charset_normalizer-3.4.1/pyproject.toml charset_normalizer-3.4.1-new/pyproject.toml
--- charset_normalizer-3.4.1/pyproject.toml	2024-12-24 17:22:27.000000000 +0100
+++ charset_normalizer-3.4.1-new/pyproject.toml	2025-02-21 13:36:31.646375614 +0100
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "setuptools-scm", "mypy>=1.4.1,<=1.14.0"]
+requires = ["setuptools", "setuptools-scm", "mypy>=1.4.1,<=1.14.10"]
 build-backend = "setuptools.build_meta"
 
 [project]
