diff -uNr ruamel.yaml.jinja2-0.2.7/__init__.py ruamel.yaml.jinja2-0.2.7-new/__init__.py
--- ruamel.yaml.jinja2-0.2.7/__init__.py	2021-09-23 17:11:56.000000000 +0200
+++ ruamel.yaml.jinja2-0.2.7-new/__init__.py	2025-03-10 23:57:56.272307657 +0100
@@ -17,7 +17,6 @@
     universal=True,
     keywords='yaml 1.2 parser round-trip jinja2',
     nested=True,
-    install_requires=['ruamel.yaml>=0.16.1'],
     python_requires='>=3.6',
     tox=dict(env='3'),
 )
