diff -uNr sqlparse-0.5.3/sqlparse/cli.py sqlparse-0.5.3-new/sqlparse/cli.py
--- sqlparse-0.5.3/sqlparse/cli.py	2020-02-02 01:00:00.000000000 +0100
+++ sqlparse-0.5.3-new/sqlparse/cli.py	2025-02-19 10:35:45.609758560 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2009-2020 the sqlparse authors and contributors
 # <see AUTHORS file>
diff -uNr sqlparse-0.5.3/sqlparse/__main__.py sqlparse-0.5.3-new/sqlparse/__main__.py
--- sqlparse-0.5.3/sqlparse/__main__.py	2020-02-02 01:00:00.000000000 +0100
+++ sqlparse-0.5.3-new/sqlparse/__main__.py	2025-02-19 10:35:37.472449061 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #
 # Copyright (C) 2009-2020 the sqlparse authors and contributors
 # <see AUTHORS file>
