diff -uNr yarl-1.9.4/packaging/pep517_backend/_backend.py yarl-1.9.4-new/packaging/pep517_backend/_backend.py
--- yarl-1.9.4/packaging/pep517_backend/_backend.py	2023-12-06 12:14:56.000000000 +0100
+++ yarl-1.9.4-new/packaging/pep517_backend/_backend.py	2024-03-30 14:29:35.880034367 +0100
@@ -286,7 +286,7 @@
     """
     with maybe_prebuild_c_extensions(
             line_trace_cython_when_unset=False,
-            build_inplace=False,
+            build_inplace=True,
             config_settings=config_settings,
     ):
         return _setuptools_build_wheel(
